scripts/x-charts-pro.exports.json

```diff
@@ -202,7 +202,6 @@
   { "name": "legendClasses", "kind": "Variable" },
   { "name": "LegendItemContext", "kind": "TypeAlias" },
   { "name": "LegendItemParams", "kind": "Interface" },
-  { "name": "LegendPosition", "kind": "TypeAlias" },
   { "name": "LineChart", "kind": "Variable" },
```

Comment:
That's a breaking change. Do you have other places where this might be useful?

If yes, the renaming could be added to the PR description, and have a small point the migration guide. Othersie someone who used it will loose sometime to know what is the new name of this type.

A codemod should not take too long thanks to the `renameImports` helper

Reply:
done